src/image_processing/command_parser/command_parser.py

```diff
@@ -1,28 +1,41 @@
 from abc import ABC, abstractmethod
-from typing import List
+from dataclasses import dataclass
+from typing import List, Optional
 
 from PIL import Image
 
 from src.image_processing.command import Command
+from src.image_processing.command_parser.language_package import LanguageType, get_language_package
+
+
+@dataclass
+class ParserParameters:
+    """
+    Parameters for command parser
+    """
+
+    num_few_shot_samples: int = -1
```

Comment:
mb set to 0?

Reply:
Could you check it? https://github.com/HabKaffee/ml_tg_bot/pull/18#discussion_r2008302121
If everything is OK, I could move to 0

Reply:
add check for max_few_shot_samples. because if more than max it crashes

Reply:
Could you provide details about crash?
If you pass `num_few_shot_samples` greater the number of `few_shot_samples` it will be defined as `len(few_shot_samples)`

src/image_processing/command_parser/ai_command_parser.py:42
```python
_num_samples = min(num_few_shot_samples, len(few_shot_samples))
```

Reply:
maybe the WSL problem, but crashes only in that case